### DISCUSSION TITLE
vc-viewer透過onlayerAdded取得的imageryLayer的屬性值怎麼取?

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#357] [Q&A]
您好,
vc-viewer透過onlayerAdded取得的imageryLayer,
透過CONSOLE.LOG後,第一層看的到imageryProvider
再看進去後,原本看到的defaultAlpha的名稱都不見,
這個imageryProvider內的值怎麼取?
謝謝.

![2023-02-18_113938](https://user-images.githubusercontent.com/5283322/219829832-64fe4a2a-0aa6-4608-889c-17335878132c.png)

**Comment by zouyaoji** [comment]
你输出这个就是imagerylayer啊。
直接改alpha就行的。
